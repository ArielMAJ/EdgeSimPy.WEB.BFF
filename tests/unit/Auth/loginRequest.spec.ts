// import { createTestClient } from "apollo-server-testing";
// import { loginRequestMutation } from "./__gql__/mutations";
// import { createTestServer } from "../../utils";

// beforeAll(async () => {});

// beforeEach(async () => {});

// afterAll(async () => {});

// describe("User Login", () => {
//   it("should log in a user with valid credentials", async () => {
//     jest
//       .spyOn(
//         require("../../../src/services/authentication/login.service"),
//         "requestToken"
//       )
//       .mockResolvedValue({
//         accessToken: "some-valid-access-token",
//         tokenType: "bearer",
//         expiresAt: "2025-05-25T16:36:24.076Z",
//       });

//     const loginInput = Ëœ;

//     const apolloServer = await createTestServer();
//     const { mutate } = createTestClient(apolloServer);

//     const res = await mutate({
//       mutation: loginRequestMutation,
//       variables: loginInput,
//     });
//     expect(res).toMatchSnapshot("User login");
//   });
// });
