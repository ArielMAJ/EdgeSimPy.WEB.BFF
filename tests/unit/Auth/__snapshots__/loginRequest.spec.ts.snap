// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Login should log in a user with valid credentials: User login 1`] = `
Object {
  "data": {
    "loginRequest": {
      "accessToken": "some-valid-access-token",
      "tokenType": "bearer",
      "expiresAt": "2025-05-25T16:36:24.076Z"
    }
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
